static/js/components/VerifyButton.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Button from 'react-bootstrap/lib/Button';
+
+class VerifyButton extends React.Component {
+  render() {
+    return <Button bsStyle="success">UPGRADE TO VERIFIED</Button>;
+  }
+}
+
+VerifyButton.PropTypes = {
+};
+
+export default VerifyButton;
```

Comment:
Probably not, it was a convenience for testing but then I changed the way these are tested. I'll inline this and the EnrollButton at the point where it's used